sdk/provisioning/Azure.Provisioning/tests/ConstructTests.cs

```diff
@@ -110,7 +110,7 @@ public void GetParametersChildConstructs(bool recursive)
             var rg2 = new ResourceGroup(childScope, "rg2");
             rg2.AssignParameter(r => r.Location, new Parameter("location"));
 
-            var expected = recursive ? 2 : 1;
+            var expected = recursive ? 3 : 2;
```

Comment:
Don't we need to update the assertion that was expecting 0 as well?

Reply:
The param is added to the keyvault's scope which is the infra which is why it went up by one but the l2 construct stays 0